src/components/Pages/Guide/GuideSectionCollection.js

```diff
@@ -60,14 +34,7 @@ function GuideSection({
         <HtmlFromAdmin title={' '} content={additionalContent} />
       </div>
       <div className="coa-GuideSection__link">
-        <a
-          href={GetUrlFromTopicsOrDepartments({
-            topics,
-            departments,
-            slug,
-          })}
-          target="_blank"
-        >
+        <a href={url} target="_blank">
```

Comment:
When is it preferable to use `<Link />` vs `<a />` ? I saw above some refactoring to use Link instead of <a /> and was curious why

Reply:
I've been trying to use <Link> as much as possible so we can utilize all the benefits of the react-static router (https://github.com/react-static/react-static/blob/v5/docs/components.md#link)

I know this is a bit different in newer versions of react-static so it can definitely be an ongoing conversation